incubator-tools/labeled_dataset_validation/labeled_dataset_validation.ipynb

```diff
@@ -0,0 +1,409 @@
+{
```

Comment:
<p>Most of these functions should be moved to the utilities file</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/document-ai-samples/pull/720/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
@holtskinner, Agreed, these are generic functions but we are not using them repeatedly any other tools. These are specific to this tool.